plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -102,6 +103,18 @@ export function WriteItForMeButtonContainer() {
 			},
 		} );
 
+	const { requestCompletion: requestShortCompletion } = useCompletion( {
+		feature: WOO_AI_PLUGIN_FEATURE_NAME,
+		onStreamMessage: ( content ) => shortTinyEditor.setContent( content ),
+		onStreamError: ( error ) =>
+			shortTinyEditor.setContent( getApiError( error ) ),
```

Comment:
It might be a good idea to move away from setting the content to the error message. I know it's a bit out of scope for this PR, but I think showing the error message separately in the UI would be better. Let's also do this for the main description generator if you agree.

Reply:
@nima-karimi Yup, I definitely agree, although I'm uncertain of how/where to display these messages in the UI. The legacy product editor feels fairly clumsy to modify in those ways. If we end up going towards the modal [as Jarek has designed](https://github.com/woocommerce/woocommerce/issues/38810) then that will give us some better options, so perhaps we should see how we bottom out on that project? I'd be up for creating an issue around this regardless.

Reply:
In my draft PR for the branding work, I'm catching errors and showing them via a snackbar. 
Specifically, using the Notices module, like so:
https://developer.wordpress.org/block-editor/reference-guides/data/data-core-notices/#createerrornotice
I think it would be nice for us to align on this approach, if it sounds good enough @jarekmorawski